RELEASE.md

```diff
@@ -1,15 +1,61 @@
-Semantic Release Workflow:
+# Semantic Release Workflow
 
-1) Ensure commit message follow the following format:
+LavaMoat follows [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html).
+This lays out part of the steps involved in drafting and publishing any of the LavaMoat packages monotonically.
 
-| Commit message                                                                                                                                                                                   | Release type               |
-| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------- |
-| `fix(pencil): stop graphite breaking when too much pressure applied`                                                                                                                             | Patch Release              |
-| `feat(pencil): add 'graphiteWidth' option`                                                                                                                                                       | Feature Release  |
-| `perf(pencil): remove graphiteWidth option`<br><br>`BREAKING CHANGE: The graphiteWidth option has been removed.`<br>`The default graphite width of 10mm is always used for performance reasons.` | Breaking Release |
+## 0. Prerequirements
 
-2) Setup env vars for versioning and publishing
-  - GH_TOKEN
-  - NPM_TOKEN
+- One package per release
+- Releases are always monotonically from current `main`.
+- Ensure your local `main` branch is up to date
+- Perform any necessary manual or local tests not covered in automatic tests
+- Review the commits since the last release: `$ git log main...$(git log --grep '${PKGDIR}/version - ${CURRENT_VERSION}' --format=tformat:%h) packages/${PKGDIR}`
+- Review the diff since the last release: `$ git diff main $(git log --grep '${PKGDIR}/version - ${CURRENT_VERSION}' --format=tformat:%h) packages/${PKGDIR}`
+- Determine the new version number based on the nature of changes
+  - Follow up with author or reviewers if the SemVer-nature of an included change is unclear before proceeding
```

Comment:
whether is clear or not is better

Reply:
Could be, but I'm not sure exactly what you're after here - could you make a suggestion of how you think this sentences can be written better? :)

The intended message here is `changes.filter(c => c.ambiguous).forEach(c => c.followUp()); proceed();`. Ideally the information needed should already be derivable from the history itself so contacting authors/past reviewers only necessary to fill in gaps.

Reply:
not critical